[2023-12-16T18:07:04.326+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_data_pipeline.extract_data manual__2023-12-16T18:07:01.673409+00:00 [queued]>
[2023-12-16T18:07:04.341+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_data_pipeline.extract_data manual__2023-12-16T18:07:01.673409+00:00 [queued]>
[2023-12-16T18:07:04.341+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-16T18:07:04.373+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_data> on 2023-12-16 18:07:01.673409+00:00
[2023-12-16T18:07:04.424+0000] {standard_task_runner.py:57} INFO - Started process 11332 to run task
[2023-12-16T18:07:04.426+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_data_pipeline', 'extract_data', 'manual__2023-12-16T18:07:01.673409+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/taxi_etl.py', '--cfg-path', '/tmp/tmpnrvy47c1']
[2023-12-16T18:07:04.429+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask extract_data
[2023-12-16T18:07:04.483+0000] {task_command.py:416} INFO - Running <TaskInstance: NYC_Taxi_data_pipeline.extract_data manual__2023-12-16T18:07:01.673409+00:00 [running]> on host 6abb98cbcbb2
[2023-12-16T18:07:04.555+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='islam' AIRFLOW_CTX_DAG_ID='NYC_Taxi_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-16T18:07:01.673409+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-16T18:07:01.673409+00:00'
[2023-12-16T18:07:04.558+0000] {logging_mixin.py:154} INFO - Cleaned dataset already exists on the file system
[2023-12-16T18:07:04.558+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-16T18:07:04.567+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=NYC_Taxi_data_pipeline, task_id=extract_data, execution_date=20231216T180701, start_date=20231216T180704, end_date=20231216T180704
[2023-12-16T18:07:04.601+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-16T18:07:04.621+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
